pydicom/valuerep.py

```diff
@@ -862,8 +886,8 @@ def __new__(
     def __init__(
         self,
         val: Union[bytes, str, "PersonName"],
-        encodings: Optional[List[str]] = None,
-        original_string: Optional[str] = None
+        encodings: Optional[Sequence[str]] = None,
+        original_string: Optional[bytes] = None
```

Comment:
PersonName could use a refactor. I think it'd be better to rename or eliminate the `original_string` kwarg, make the `PersonName.original_string` attr private (`_encoded_value` or something) and provide a property for public access.

There's also [this ](https://github.com/pydicom/pydicom/blob/c73451668d9d2bf021bbec4067e31fb25083d582/pydicom/values.py#L465), where a `PersonName` is created just so `decode()` can be called which creates a new `PersonName` with the decoded value. It seems like it'd make more sense for PN to manage changes in-place (including changes to encodings and always having decoded/encoded values available).